modules/gapi/test/infer/gapi_infer_ie_test.cpp

```diff
@@ -1452,7 +1452,7 @@ TEST(Infer, TestStreamingInfer)
     cv::GMat age, gender;
 
     std::tie(age, gender) = cv::gapi::infer<AgeGender>(in);
-    cv::GComputation comp(cv::GIn(in), cv::GOut(age, gender));
+    cv::GComputation comp(cv::GIn(in), cv::GOut(age, gender, cv::gapi::ie::latency(age)));
```

Comment:
red flag here too.

Who said it will be IE if one day we swap `ie::Params` with `onnx::Params`? What about jumping to MX?

Reply:
1. The case you are speaking about is more generic and should measured as the time for all `Island` execution
2. I believe every inference island works different, and time measurement will be depend on that